rules/order.js

```diff
@@ -85,15 +174,23 @@ module.exports = {
         'line-height',
         'color',
         'text-align',
-        'white-space',
-        'vertical-align',
```

Comment:
It's a little strange to me that `vertical-align` was moved, but `line-height` wasn't. Those two are highly related, and should probably live together (either here or up in the higher block).